deploy/helm/clickhouse-operator/values.yaml

```diff
@@ -755,21 +755,21 @@ additionalResources: []
 #    kind: ConfigMap
 #    metadata:
 #      name: {{ include "altinity-clickhouse-operator.fullname" . }}-cm
-#      namespace: {{ .Release.Namespace }}
```

Comment:
Actually, I don't understand, what this kind of changes actually means?

Where did you define `altinity-cloud-opeator.namespace`?